src/typed-method-types/workflows/triggers/webhook.ts

```diff
@@ -11,3 +18,20 @@ export type WebhookTrigger =
       filter?: FilterType;
     };
   };
+export type WebhookTriggerResponse<TriggerDefinition extends WebhookTrigger> =
+  Promise<
+    WebhookResponse<TriggerDefinition> | FailedTriggerResponse
+  >;
+
+export type WebhookResponse<TriggerDefinition extends WebhookTrigger> =
+  & BaseResponse
+  & {
+    trigger: WebhookTriggerObject<TriggerDefinition>;
+  };
+
+export type WebhookTriggerObject<TriggerDefinition extends WebhookTrigger> =
+  & BaseTriggerResponse<TriggerDefinition>
+  & {
+    webhook?: TriggerDefinition["webhook"];
+    webhook_url: string;
```

Comment:
We need to figure out why `webhook_url` isn't being returned in certain cases before we can merge this PR